### DISCUSSION TITLE
Add the possibility to add/modify the priority level of the ticket

### DISCUSSION THREAD

**Discussion by PilaScat** [coderflexx/laravel-ticket#33] [Ideas]
Maybe I want more than 3 levels, like critical

**Comment by ousid** [comment]
@PilaScat 

That's totally fine, you can register any priority level, and use `withPriority` to get it.

See: https://github.com/coderflexx/laravel-ticket#ticket-scopes

**Comment by PilaScat** [comment]
Yep I found it the other day, thanksFilippo Scatigna On 28 May 2023 12:18, Oussama ***@***.***> wrote:
@PilaScat
That's totally fine, you can register any priority level, and use withPriority to get it.
See: https://github.com/coderflexx/laravel-ticket#ticket-scopes
